app/form/field/YearField.js

```diff
@@ -1,7 +1,8 @@
 Ext.define('CpsiMapview.form.field.YearField', {
```

Comment:
@leonardospina - I think there is a problem with your branch - this file keeps getting reverted on each pull request. Probably best to branch again from compassinformatics/master ?